tools/distpartitioning/data_shuffle.py

```diff
@@ -586,6 +586,40 @@ def read_dataset(rank, world_size, id_lookup, params, schema_map):
 
     return node_tids, node_features, node_feat_tids, edge_data, edge_features, edge_tids, edge_feat_tids
 
+def reorder_data(params, world_size, data, key):
+    """
+    Auxiliary function used to sort node and edge data for the input graph.
+
+    Parameters:
+    -----------
+    params : argparser object 
```

Comment:
seems only `num_parts` is needed so how about passing it directly instead of the whole `params`.

Reply:
Done.